recipe/conda_build_config.yaml

```
@@ -1,6 +1,6 @@
 CONDA_BUILD_SYSROOT:       # [osx and x86]
-  - /opt/MacOSX10.14.sdk   # [osx and x86]
+  - /opt/MacOSX10.15.sdk   # [osx and x86]
 macos_min_version:         # [osx and x86]
-  - 10.14                  # [osx and x86]
+  - 10.15                  # [osx and x86]
 MACOSX_DEPLOYMENT_TARGET:  # [osx and x86]
-  - 10.14                  # [osx and x86]
\ No newline at end of file
+  - 10.15                  # [osx and x86]
```

```suggestion
  - 10.15                  # [osx and x86]
c_compiler:                   # [win]
  - vs2022                   # [win]
cxx_compiler:               # [win]
  - vs2022                 # [win]
```
Since you have bazel configuration pointing to vs2022.